### DISCUSSION TITLE
Weird FlareSolverrSharp behavior

### DISCUSSION THREAD

**Discussion by iwannadiehard** [FlareSolverr/FlareSolverr#1125] [Q&A]
Yesterday i faced an issue while web scraping of cloudflare and after some tinkering decided to use FlareSolverSharp to aid me.
I did nothing but run command Install-Package FlareSolverr and configure htmlagilitypack to get the handler as in the example.
Then right after running the program cloudflare was solved and i had no trouble, scarping website i wanted and developing programm further.

Today no matter what i did it displays it threw "HTTP StatusCode not 200 or 500. Status is :NotFound
   at FlareSolverrSharp.Solvers.FlareSolverr" at me up until i downloaded and started the flaversolver. Also it now takes around 30 seconds to solve the cloudflare, b4 that it solved everything in less than a second (it now gives cookies error, but it's not the point).
   
Question is: What could've caused such behavior? Why did it work without running the .exe file before? Same thing happened with htmlagilitypack and handlers, first day it worked without user-agent at all and now i need user-agent or i server doesn't let me in.

**Comment by ilike2burnthing** [comment]
FlareSolverr needs to be running to solve challenges. No Cloudflare challenge is solved in 'less than second', so I'm not really sure what you were doing yesterday.
